front/app/routes/builds.js

```diff
@@ -0,0 +1,88 @@
+import Ember from 'ember';
+import fetch from 'fetch';
+import ENV from 'rust-dashboard/config/environment';
+
+function fixTimestamps(data) {
+  return data.map(elt => {
+    return [elt[0] * 1000, elt[1]];
+  });
+}
+
+function getDisplayName(build) {
+  // TODO: Use better names
+  if (build.name === "buildbot") {
+    return build.env;
+  } else if (build.name === "travis") {
+    return build.env;
+  } else if (build.name === "appveyor") {
+    return build.env;
+  }
```

Comment:
For right now, I just used the build environment as the display name -- we can always iterate on more readable names separately.